### DISCUSSION TITLE
Latest Pull Not Updated

### DISCUSSION THREAD

**Discussion by DBtheDBA** [jlesage/docker-crashplan-pro#453] [General]
Just checking on if it was intentional not to rebuild the latest image for pull?
jlesage/crashplan-pro   latest    (id)   **3 weeks ago**    609MB
Looks like it was updated but not yet published.  Thanks much.

**Comment by jlesage** [comment]
A new image is now available!

**Reply by DBtheDBA** [reply]
Thank you!
